test/beacon/pages_test.exs

```diff
@@ -0,0 +1,69 @@
+defmodule Beacon.PagesTest do
+  use Beacon.DataCase, async: true
+
+  alias Beacon.Pages
+  alias Beacon.Layouts
+  alias Beacon.Components
+  alias Beacon.Stylesheets
+  alias Beacon.Pages.Page
+
+  defp create_page do
```

Comment:
just copied it over from Readme to get the first example running. The focus of this PR is the setup, so not sure if we already want to do something more elaborated here.